edx_proctoring/static/proctoring/js/dropdown.js

```diff
@@ -0,0 +1,84 @@
+edx = edx || {};
+
+(function($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+    edx.dashboard.dropdown = {};
+
+    edx.dashboard.dropdown.toggleExamAttemptActionDropdownMenu = function(event) {
+        var $target = $(event.currentTarget),
+            dashboardIndex = $target.data().dashboardIndex,
+            $dropdown = $($target.data('dropdownSelector') || '#actions-dropdown-' + dashboardIndex),
+            $dropdownButton = $($target.data('dropdownButtonSelector') || '#actions-dropdown-link-' + dashboardIndex),
+            ariaExpandedState = ($dropdownButton.attr('aria-expanded') === 'true'),
+            menuItems = $dropdown.find('a');
+
+        var catchKeyPress = function(object, keyPressEvent) {
+            // get currently focused item
+            var $focusedItem = $(':focus');
+
+            // get the index of the currently focused item
+            var focusedItemIndex = menuItems.index($focusedItem);
+
+            // var to store next focused item index
+            var itemToFocusIndex;
+
+            // if space or escape key pressed
+            if (keyPressEvent.which === 32 || keyPressEvent.which === 27) {
```

Comment:
#NIT can you put the number `32` and `27` into class member variables with name of the keys? It makes the code easier to read.